pkgs/by-name/op/openfga/package.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "openfga";
+  version = "1.5.0";
+
+  src = fetchFromGitHub {
+    owner = pname;
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-lqAoIOnizfvZm/NjGeinXzoci+ykmzdFtLDGeNoi97c=";
+  };
+
+  vendorHash = "sha256-vUfyo7/mapAzNs6RV+GVj5gOPEpMypNNIkoUgnBYTjA=";
+
+  subPackages = [ "cmd/${pname}" ];
+
+  meta = with lib; {
+    homepage = "https://github.com/openfga/openfga";
+    description = "A high-performance and flexible authorization/permission engine built for developers and inspired by Google Zanzibar";
+    license = licenses.asl20;
+    maintainers = with maintainers; [ lucperkins ];
```

Comment:
Can you add `mainProgram` and `platforms`?

Reply:
Done